tests/collections/asr/test_asr_datasets.py

```diff
@@ -13,16 +13,46 @@
 # limitations under the License.
 
 import copy
+import json
 import os
+import tempfile
 
 import pytest
+import torch.cuda
 from omegaconf import OmegaConf
 
 from nemo.collections.asr.data.audio_to_text import TarredAudioToBPEDataset, TarredAudioToCharDataset
+from nemo.collections.asr.data.audio_to_text_dali import AudioToBPEDALIDataset, AudioToCharDALIDataset
 from nemo.collections.asr.data.audio_to_text_dataset import inject_dataloader_value_from_model_config
 from nemo.collections.common import tokenizers
 from nemo.utils import logging
 
+try:
+    import nvidia.dali as dali
```

Comment:
As discussed earlier, we should probably check that DALI is at the version 1.4 or later.